e2e/cypress/e2e/specs/checkout/checkout-cost-center-order.b2b.e2e-spec.ts

```diff
@@ -1,56 +1,84 @@
 import { at } from '../../framework';
+import { createB2BUserViaREST } from '../../framework/b2b-user';
 import { LoginPage } from '../../pages/account/login.page';
 import { sensibleDefaults } from '../../pages/account/registration.page';
 import { CartPage } from '../../pages/checkout/cart.page';
 import { CheckoutPaymentPage } from '../../pages/checkout/checkout-payment.page';
 import { CheckoutReceiptPage } from '../../pages/checkout/checkout-receipt.page';
 import { CheckoutReviewPage } from '../../pages/checkout/checkout-review.page';
+import { HomePage } from '../../pages/home.page';
+import { CostCenterBuyersPage } from '../../pages/organizationmanagement/cost-center-buyers.page';
+import { CostCenterCreatePage } from '../../pages/organizationmanagement/cost-center-create.page';
+import { CostCenterDetailPage } from '../../pages/organizationmanagement/cost-center-detail.page';
+import { CostCentersPage } from '../../pages/organizationmanagement/cost-centers.page';
 import { ProductDetailPage } from '../../pages/shopping/product-detail.page';
 
 const _ = {
   costCenter: {
-    id: '100401',
+    costCenterId: '100401',
     name: 'Oil Corp Subsidiary 1',
+    budgetValue: 999,
+    budgetPeriod: 'monthly',
   },
   user: {
-    login: 'bboldner@test.intershop.de',
-    password: sensibleDefaults.password,
+    login: `test${new Date().getTime()}@testcity.de`,
+    ...sensibleDefaults,
```

Comment:
Reorder the object spread parameter. When a login property will be assigned to sensibleDefaults, then the given value will be overwritten.